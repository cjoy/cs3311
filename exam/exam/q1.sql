-- COMP3311 12s1 Exam Q1
-- The Q1 view must have attributes called (team,matches)

drop view if exists Q1;
create view Q1
as
... SQL code for view goes here ...
;
