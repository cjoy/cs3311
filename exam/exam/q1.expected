Argentina|54
Australia|50
Brazil|60
China|53
England|48
France|52
Germany|56
Iran|49
Italy|38
Japan|61
Korea|46
Netherlands|53
Spain|64
Sweden|67
Uruguay|49
